Source/monster.cpp

```diff
@@ -50,6 +50,19 @@ void NewMonsterAnim(int i, AnimStruct *anim, Direction md, AnimationDistribution
 	monst->_mdir = md;
 }
 
+void StartMonsterGotHit(int mid)
+{
+	if (Monsters[mid].MType->mtype != MT_GOLEM) {
+		NewMonsterAnim(mid, &Monsters[mid].MType->Anims[MA_GOTHIT], Monsters[mid]._mdir, gGameLogicStep < GameLogicStep::ProcessMonsters ? AnimationDistributionFlags::ProcessAnimationPending : AnimationDistributionFlags::None);
+		Monsters[mid]._mmode = MM_GOTHIT;
+	}
+	Monsters[mid].position.offset = { 0, 0 };

```

Comment:
Should we consider creating a static field `Displacement::None` or `Displacement::Zero` for readability?

Reply:
Currently Direction::DIR_OMNI converts to a Displacement of 0, 0 however I don't want to suggest using that enum value here. In this context I think a separate static for 0 magnitude displacement as you suggest makes more sense.

Reply:
I also think this is a good idea. 🙂 
But as this is used in another places too, this should be in a separate refactoring pr.